### ISSUE TITLE
Modify/install new SSH key is broken

### ISSUE THREAD

**Issue by ibaldin** [RENCI-NRIG/orca5#169]
nova-essex-prepare-key requires knowing AMI/AKI and ARI which are supplied only at instance creation and not during modify (script used for both). Makes it fail on modify install new SSH key.

**Comment by ibaldin** [comment]
Can we find a way to deploy this on a test rack so we can be sure it works? @mcevik0 @hinchliff before we cut the new release?

**Comment by ibaldin** [comment]
Addressed in 2f1e845eb115786b4ce3435060811ebede552ca1

**Comment by mcevik0** [comment]
Deployed to GWU and tested. Works as expected.
```
./modifySliverSSH.py  -i "mcevik-gwu-1" -r "493a98b0-b41f-4fc9-8230-ea211394f9bb" -s https://geni.renci.org:11443/orca/xmlrpc -c ~/.ssl/geni-mcevik.pem -p ~/.ssl/geni-mcevik.pem -k ~/.ssh/id_rsa.pub -u mcevik1
User mcevik1 key ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC/t33Xsbeu/O7rZezlexJrv4LD7Wrz+CdWEaSUcEqDC/jlaBFAUD5voj+tNoL14sa9ee2TdtcAaX5JfuccNqK+X950QCkid+268QayeVRlsX38ZWE+H5RiZmhePSckAEB4hasP34REu+BT2QWUmeaPxy1EW1v86wIDqKX9sPw+o5pLUzruTo/1n+iVPU47WQmSwh/kFJ3CTg764wE0k9ErUIlFDNl+ZHvW4C9N9cjoViOB/EZydwASbtNa2ZifCtx+7PqcmLFK1EPbGQ7LzJ0wAEeGCf2kcuH1aonR42NNB7ByjfPMpWJTE73xJNGxzhNrLXBBqQ6RYlK630weUQfl mcevik@renci.org

Modifying SSH keys for reservation ... 

{'err': False, 'ret': True}
```

VM on GWU:
```
-bash-4.1# cat ~mcevik1/.ssh/authorized_keys 
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC/t33Xsbeu/O7rZezlexJrv4LD7Wrz+CdWEaSUcEqDC/jlaBFAUD5voj+tNoL14sa9ee2TdtcAaX5JfuccNqK+X950QCkid+268QayeVRlsX38ZWE+H5RiZmhePSckAEB4hasP34REu+BT2QWUmeaPxy1EW1v86wIDqKX9sPw+o5pLUzruTo/1n+iVPU47WQmSwh/kFJ3CTg764wE0k9ErUIlFDNl+ZHvW4C9N9cjoViOB/EZydwASbtNa2ZifCtx+7PqcmLFK1EPbGQ7LzJ0wAEeGCf2kcuH1aonR42NNB7ByjfPMpWJTE73xJNGxzhNrLXBBqQ6RYlK630weUQfl mcevik@renci.org
```
